pkg/services/auth/idimpl/signer.go

```diff
@@ -37,7 +37,7 @@ func (s *LocalSigner) SignIDToken(ctx context.Context, claims *auth.IDClaims) (s
 		return "", err
 	}
 
-	builder := jwt.Signed(signer).Claims(claims)
+	builder := jwt.Signed(signer).Claims(&claims.Rest).Claims(claims.Claims)
```

Comment:
Just to make sure I get this straight. This flattens the claims and thus provides the same ID token as before right?

Reply:
yes it does